cpplib/Tests/GeometryTest.cpp

```
@@ -783,211 +784,229 @@ TEST_F(PolygonTest, EdgeCases) {
-		{3,4}
-	};
-
-	vd.calculateFaces<int>(bondlist);
-	cells = vd.extractCells();
+using namespace cpplib;
 
-	for (const auto& cell : cells) {
-		validateVoronoiCell(cell);
-	}
-}
+TEST(Voronoi, CellConstruction) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)